modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -256,7 +256,7 @@ export class BitGoAPI implements BitGoBase {
     this._validate = params.validate === undefined ? true : params.validate;
 
     if (!params.hmacVerification && params.hmacVerification !== undefined) {
-      if (common.Environments[env].hmacVerificationEnforced) {
+      if ((env == 'prod' || env == 'adminProd') && common.Environments[env].hmacVerificationEnforced) {
```

Comment:
we should also change the value of `hmacVerificationEnforced` for testnetBase to false over in https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/environments.ts#L132